sal_interfaces/ESS/ESS_Telemetry.xml

```diff
@@ -292,4 +292,82 @@
       <Count>1</Count>
     </item>
   </SALTelemetry>
+  <SALTelemetry>
+    <EFDB_Topic>ESS_hx85a</EFDB_Topic>
+    <Description>Omega HX85A Humidity Sensors</Description>
+    <item>
+      <EFDB_Name>sensorName</EFDB_Name>
+      <Description>The name of the sensor.</Description>
+      <IDL_Type>string</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>timestamp</EFDB_Name>
+      <Description>Time at which the data was determined (TAI unix seconds).</Description>
+      <IDL_Type>double</IDL_Type>
+      <Units>second</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>relativeHumidity</EFDB_Name>
+      <Description>Relative Humidity</Description>
+      <IDL_Type>float</IDL_Type>
+      <Units>%</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>airTemperature</EFDB_Name>
+      <Description>Air Temperature</Description>
+      <IDL_Type>float</IDL_Type>
+      <Units>deg_C</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>dewPoint</EFDB_Name>
+      <Description>Dew Point</Description>
+      <IDL_Type>float</IDL_Type>
+      <Units>deg_C</Units>
+      <Count>1</Count>
+    </item>
+  </SALTelemetry>
+  <SALTelemetry>
+    <EFDB_Topic>ESS_hx85ba</EFDB_Topic>
+    <Description>Omega HX85BA Humidity Sensors</Description>
```

Comment:
Please consider "Humidity, Temperature, and Pressure Sensor" or similar

Reply:
I changed it to `Omega HX85BA Humidity, Temperature and Pressure Sensor`.